### DISCUSSION TITLE
Score V2 recalculations?

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#23761] [Q&A] [answered]
Are recalculations from the old lazer scoring to Score V2 planned? Currently scores set before the most recent update have a massive advantage (one of my 96% acc FCs with SV2 is about 75,000 points behind an old FC with 91%).

**Comment by peppy** [comment, marked as answer]
We'll probably just reset then instead. Further work is being done on the conversion process (for local scores / score v1) so stay tuned for that.

**Reply by Non-Binary-Programmer** [reply]
Is there a planned date for a leaderboard reset, or is it contingent on the score conversion process being finished?

**Reply by peppy** [reply]
When we get to it, basically. Sorry I can't give a more solid answer.
